custom/enikshay/tasks.py

```diff
@@ -86,10 +88,12 @@ def run(self):
                 adherence_update = EpisodeAdherenceUpdate(episode, self)
                 voucher_update = EpisodeVoucherUpdate(self.domain, episode)
                 test_update = EpisodeTestUpdate(self.domain, episode)
+                episode_facility_id_migration = EpisodeFacilityIDMigration(self.domain, episode)
```

Comment:
Isn't this a one time migration, why not put this in a management command instead of a periodic task that repeats?